dart/lib/src/protocol/sentry_id.dart

```diff
@@ -0,0 +1,16 @@
+/// Sentry response id
+
+const _dashIndexed = [8, 13, 18, 23];
+
+class SentryId {
+  static final String emptyId =
```

Comment:
I'd propose hard code the `0000` here instead of running this code. It's a `const`

Reply:
✅